src/wp/SplashScreen.cs

```diff
@@ -163,5 +190,30 @@ public void hide(string options = null)
                 story.Begin();
             });
         }
+
+        private void StartAutoHideTimer()
+        {
+            var timer = new DispatcherTimer() { Interval = TimeSpan.FromMilliseconds(preferences.SplashScreenDelay) };
+            timer.Tick += (object sender, EventArgs e) =>
+            {
+                this.hide();
+                timer.Stop();
+            };
+            timer.Start();
+        }
+
+        private class Preferences
```

Comment:
It's not clear that another heap-allocated object is worth it here.  With only three preferences, in a private class, each of these could be promoted to a member variable of the owning class, eliminating the private class altogether.